src/main/java/mod/acgaming/universaltweaks/mods/actuallyadditions/dupes/mixin/StackUtilMixin.java

```diff
@@ -0,0 +1,40 @@
+package mod.acgaming.universaltweaks.mods.actuallyadditions.dupes.mixin;
+
+import net.minecraft.item.ItemStack;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+
+import com.llamalad7.mixinextras.injector.wrapoperation.Operation;
+import com.llamalad7.mixinextras.injector.wrapoperation.WrapOperation;
+
+import de.ellpeck.actuallyadditions.mod.util.ItemStackHandlerAA;
+import de.ellpeck.actuallyadditions.mod.util.StackUtil;
+
+/**
+ * Fixes `canAddAll` function increasing stack counts in stack list;
+ * under specific edge cases (e.g. inventory is empty)
+ */
+@Mixin(value = StackUtil.class, remap = false)
+public class StackUtilMixin {
+
+    @WrapOperation(method = "canAddAll(Lde/ellpeck/actuallyadditions/mod/util/ItemStackHandlerAA;Ljava/util/List;Z)Z",
```

Comment:
i believe you could combine these two mixins into one via
```suggestion
    @WrapOperation(method = {
        "canAddAll(Lde/ellpeck/actuallyadditions/mod/util/ItemStackHandlerAA;Ljava/util/List;Z)Z",
        "canAddAll(Lde/ellpeck/actuallyadditions/mod/util/ItemStackHandlerAA;Ljava/util/List;IIZ)Z",
    },
```